BGAnimations/ScreenCRTTestPatterns underlay/default.lua

```diff
@@ -0,0 +1,77 @@
+local patterns = {
+	{name=Screen.String("SMPTEColorBars"), file="smpte-color-bars.gif"},
+	{name=Screen.String("Convergence"), file="convergence.gif"},
+	{name=Screen.String("Gamma"), file="gamma.jpg"},
+	{name=Screen.String("AspectRatio"), file="aspect-ratio.jpg"},
+}
+
+local idx = 1
+
+local function InputHandler(event)
+	if event.type == "InputEventType_FirstPress" then
+		if event.GameButton == "Back" then
```

Comment:
What about Start instead of Back? Not everyone has Back mapped on a machine (or even a keyboard connected).

Reply:
updated to allow both buttons